openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/AppRepository.java

```diff
@@ -164,12 +164,12 @@ public void storeRelationships(App entity) {
     }
   }
 
-  public final List<AppRunRecord> listAll() {
+  public final List<App> listAll() {
     // forward scrolling, if after == null then first page is being asked
     List<String> jsons = dao.listAfterWithOffset(Integer.MAX_VALUE, 0);
-    List<AppRunRecord> entities = new ArrayList<>();
+    List<App> entities = new ArrayList<>();
     for (String json : jsons) {
-      AppRunRecord entity = JsonUtils.readValue(json, AppRunRecord.class);
+      App entity = JsonUtils.readValue(json, App.class);
```

Comment:
hi @mohityadav766 this method was not used anywhere, but I think it was casting to the wrong class. Fixed it and using it to init apps on server start

Reply:
that's correct @pmbrull